src/functions/template-tags/html.php

```diff
@@ -225,3 +225,46 @@ function tribe_format_field_dependency( $deps ) {
 
 	return $dependency;
 }
+
+if ( ! function_exists( 'tec_embed_header' ) ) {
+	/**
+	 * Generic Iframe header for frontend use.
+	 *
+	 * @since TBD
+	 */
+	function tec_embed_header(): void {
+		show_admin_bar( false ); //phpcs:ignore WordPressVIPMinimum.UserExperience.AdminBarRemoval.RemovalDetected
+
+		header( 'Content-Type: ' . get_option( 'html_type' ) . '; charset=' . get_option( 'blog_charset' ) );
+		?>
+		<!DOCTYPE html>
+		<html <?php language_attributes(); ?>>
+			<head>
+				<title><?php wp_title(); ?></title>
+				<meta charset="<?php bloginfo( 'charset' ); ?>">
+				<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+				<link rel="profile" href="https://gmpg.org/xfn/11">
+
+				<?php wp_head(); ?>
+			</head>
+			<body <?php body_class(); ?>>
+				<?php wp_body_open(); ?>
+		<?php
```

Comment:
Throw this into a template, so it's filterable.

Reply:
Done!